pyspider/fetcher/tornado_fetcher.py

```diff
@@ -64,7 +64,7 @@ def size(self):
 
 
 class Fetcher(object):
-    user_agent = "pyspider/%s (+http://pyspider.org/)" % pyspider.__version__
+    user_agent = "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/28.0.1500.52 Safari/537.36"
```

Comment:
There is a UA option for fetcher: http://docs.pyspider.org/en/latest/Command-Line/#fetcher